src/components/GlobalNavigation/styles.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import palette from '../../theme/palette';
+
+const NavBar = styled.nav`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
```

Comment:
IMO, `width: 100%;` looks better than `left: 0; right: 0;`.

You don't have to resolve this.